src/config/backends/extlinux-file.ts

```diff
@@ -0,0 +1,86 @@
+import * as _ from 'lodash';
+
+import { ConfigOptions } from './backend';
+import { ExtLinuxParseError } from '../../lib/errors';
+
+export interface ExtlinuxFile {
+	labels: Label;
+	globals: Directive;
+}
+
+export interface Label {
+	[labelName: string]: Directive;
+}
+
+export interface Directive {
+	[directive: string]: string;
```

Comment:
We could more strongly type this, right? 
```
export interface Directive {
    [key in 'APPEND' | 'FDT']: string
```

Reply:
I have been able to achieve a stronger typing for Directive but it actually just complicated things more. I may need some help to figure out a way to do this.

Reply:
We agreed to move on from this for now as there's no decent solution I liked.